turf/rhumb_destination/tests/test_rhumb_destination.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import os
+
+from turf.helpers import line_string, feature_collection
+from turf.invariant import get_coords_from_features
+from turf.rhumb_destination import rhumb_destination
+
+from turf.utils.error_codes import error_code_messages
+from turf.utils.exceptions import InvalidInput
+from turf.utils.test_setup import get_fixtures
+
+current_path = os.path.dirname(os.path.realpath(__file__))
+
+fixtures = get_fixtures(current_path)
+
+
+class TestRhumbDestination:
+    @pytest.mark.parametrize(
+        "fixture",
+        [
+            pytest.param(fixture, id=fixture_name)
+            for fixture_name, fixture in fixtures.items()
+        ],
+    )
+    def test_rhumb_destination(self, fixture):
+
+        coords = get_coords_from_features(fixture["in"])
```

Comment:
I dont think we should be retrieving the coords from the feature before passing on to `rhumb_destination`, as it should be tested that `rhumb_destination` can accept geojson features. You could potentially add a separate test to check if `rhumb_destination` can also accept a list as input.